hudi-sync/hudi-hive-sync/src/test/java/org/apache/hudi/hive/testutils/HiveTestUtil.java

```diff
@@ -176,6 +178,16 @@ public static void createCOWTable(String instantTime, int numberOfPartitions, bo
     createCommitFile(commitMetadata, instantTime);
   }
 
+  public static void createReplaceCommit(String instantTime, String partitions)
+      throws IOException {
+    HoodieReplaceCommitMetadata replaceCommitMetadata = new HoodieReplaceCommitMetadata();
+    replaceCommitMetadata.setOperationType(WriteOperationType.DELETE_PARTITION);
```

Comment:
Here as a method to pass parameters? 
createReplaceCommit(String instantTime, String partitions) ->  createReplaceCommit(String instantTime, String partitions, WriteOperationType OperationType)

Reply:
Sure thing, Changed. Thanks a lot for your review.